admin/src/scenes/volontaires/list.js

```diff
@@ -478,11 +479,11 @@ const Hit = ({ hit, onClick, selected }) => {
           </p>
         </MultiLine>
       </td>
-      <td>
-        <Badge minify text={hit.cohort} tooltipText={`Cohorte ${hit.cohort}`} />
-        <BadgePhase text="Phase 1" value={hit.statusPhase1} />
-        <BadgePhase text="Phase 2" value={hit.statusPhase2} />
-        <BadgePhase text="Phase 3" value={hit.statusPhase3} />
+      <td onClick={(e) => e.stopPropagation()}>
+        <Badge minify text={hit.cohort} tooltipText={`Cohorte ${hit.cohort}`} style={{ cursor: "default" }} />
+        <BadgePhase text="Phase 1" href="phase1" value={hit.statusPhase1} id={hit._id} />
```

Comment:
pourquoi ne pas faire :
```suggestion
        <BadgePhase text="Phase 1" redirect={`/volontaire/${hit._id}/phase1`} value={hit.statusPhase1} />
```

une seule prop, bien explicite ici.
j'aime pas `href` en fait, le truc qui existe déjà pour des éléments html ca embrouille je trouve, non ?